client/my-sites/site-settings/section-import.jsx

```diff
@@ -16,31 +18,26 @@ import MediumImporter from 'my-sites/importer/importer-medium';
 import { fetchState } from 'lib/importer/actions';
 import { appStates, WORDPRESS, MEDIUM } from 'state/imports/constants';
 import EmailVerificationGate from 'components/email-verification/email-verification-gate';
+import { getSelectedSite } from 'state/ui/selectors';
 
-export default React.createClass( {
-	displayName: 'SiteSettingsImport',
+class SiteSettingsImport extends Component {
+	static propTypes = {
+		site: PropTypes.object,
```

Comment:
That makes complete sense, of course. But I've done this mainly for consistency reasons - we're using `PropTypes.object` for nearly all cases where we expect a `site` object throughout Calypso. And passing a shape of single `ID` does not help too much IMHO - the shape can vary in separate `shape` usages based on the component needs, and that might confuse someone that we're expecting different forms of `site`. I think in the ideal case we'd have our own custom `site` propType, which would contain at minimum the required site object attributes. But until then, I think `object` is good enough.